### PR TITLE
CAMEL-11596:Fix to get info about a single route we need to

### PR DESCRIPTION
**Pull request by ramu11**
[CAMEL-11596] Fix to get info about a single route we need to

### CODE REVIEW

**Comment by ramu11** [comment]
Could somebody verify

File: components/camel-spring-boot/src/main/java/org/apache/camel/spring/boot/actuate/endpoint/CamelRoutesMvcEndpoint.java [hunk 1]
```diff
@@ -60,7 +60,7 @@ public CamelRoutesMvcEndpoint(CamelRoutesEndpoint delegate) {
 
     @ResponseBody
     @GetMapping(
-        value = "/{id}/info",
+        value = {"/{id}/info", "/{id}/detail"},
```
Comment (lburgazzoli) [inline components/camel-spring-boot/src/main/java/org/apache/camel/spring/boot/actuate/endpoint/CamelRoutesMvcEndpoint.java]: I think we should not have two paths serving the same purpose so I'd go for detail only

File: components/camel-spring-boot/src/main/java/org/apache/camel/spring/boot/actuate/endpoint/CamelRoutesMvcEndpoint.java [hunk 2]
```diff
@@ -78,6 +78,27 @@ public Object info(
...
+    @ResponseBody
+    @GetMapping(
+        value = "/{id}",
+        produces = {
+            ActuatorMediaTypes.APPLICATION_ACTUATOR_V1_JSON_VALUE,
+            MediaType.APPLICATION_JSON_VALUE
+        }
+    )
+    public Object routeInfo(
+            @PathVariable String id) {
```
Comment (lburgazzoli) [inline components/camel-spring-boot/src/main/java/org/apache/camel/spring/boot/actuate/endpoint/CamelRoutesMvcEndpoint.java]: maybe this method should be just named ``info`` ? and the previous one for detailed report ``detail``

Comment (ramu11) [inline components/camel-spring-boot/src/main/java/org/apache/camel/spring/boot/actuate/endpoint/CamelRoutesMvcEndpoint.java]: Thanks   lburgazzoli  ,I will do changes as per your suggestions

**Comment by lburgazzoli** [comment]
Can you squash the commits and amend it with the same title as the issue ?

**Comment by lburgazzoli** [comment]
I see that the body of the commit message contains some noise like

```
CAMEL-11596:Fix to get info about a single route we need to
CAMEL-11596:Fix to get info about a single route we need to
CAMEL-11596:Fix to get info about a single route we need to
CAMEL-11596:Removed Unused Imports
CAMEL-11596: Implemented Reviewer suggestions
"CAMEL-11596:Spring-boot - actuator endpoint routes - Get single route only"
```

Can we get rid of such noise ?

**Comment by davsclaus** [comment]
There is also #1858 which changes in similar code, so we may get merge conflicts.

**Comment by ramu11** [comment]
So this change has to be done after merging the 1858 PR . Please let me know

**Comment by ramu11** [comment]
Seems like PR got messed up . Closing this to raise new one

**Pull request closed without merging**